hedera-node/test-clients/src/main/java/com/hedera/services/bdd/suites/leaky/LeakyCryptoTestsSuite.java

```diff
@@ -979,13 +983,22 @@ final HapiSpec contractCallAfterEthereumTransferLazyCreate() {
                 }));
     }
 
+    @HapiTest
     final HapiSpec lazyCreateViaEthereumCryptoTransfer() {
         final var RECIPIENT_KEY = LAZY_ACCOUNT_RECIPIENT;
         final var lazyCreateTxn = PAY_TXN;
         final var failedLazyCreateTxn = "failedLazyCreateTxn";
         return propertyPreservingHapiSpec("lazyCreateViaEthereumCryptoTransfer")
                 .preserving(CHAIN_ID_PROP, LAZY_CREATE_PROPERTY_NAME, CONTRACTS_EVM_VERSION_PROP)
                 .given(
+                        snapshotMode(
+                                FUZZY_MATCH_AGAINST_HAPI_TEST_STREAMS,
+                                ALLOW_SKIPPED_ENTITY_IDS,
+                                HIGHLY_NON_DETERMINISTIC_FEES), // TODO: even though adding this makes fuzzy matching to
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**question:** @Neeharika-Sompalli  ‏@tinker-michaelj do you have some ideas what might be causing such big difference in transaction fees for those cases?

Reply:
@MiroslavGatsanoga  I have a PR open for fixing almost except once, `LeakyCryptoTestsSuite` https://github.com/hashgraph/hedera-services/pull/10500. Fixed the issue there.

Reply:
EDIT: Sorry Miro, this is just enabling HapiTest. Not fuzzy matching. 
@tinker-michaelj Could you please check if we are charging wrong fee when we throw `INSUFFICIENT_GAS` in modular code?